docs/developer/contributing.rst

```diff
@@ -35,4 +67,63 @@ Notes:
 3) Installing ``pandoc`` allows you to verify that automatic rendering of Jupyter notebooks
    into documentation for ReadTheDocs works as expected. For more information, see
    the Python Project Template documentation on
-   `Sphinx and Python Notebooks <https://lincc-ppt.readthedocs.io/en/stable/practices/sphinx.html#python-notebooks>`_.
\ No newline at end of file
+   `Sphinx and Python Notebooks <https://lincc-ppt.readthedocs.io/en/stable/practices/sphinx.html#python-notebooks>`_.
+
+.. tip::
+    Installing on Mac
+
+    Native prebuilt binaries for healpy on Apple Silicon Macs
+    `do not yet exist <https://healpy.readthedocs.io/en/latest/install.html#binary-installation-with-pip-recommended-for-most-other-python-users>`_,
+    so it's recommended to install via conda before proceeding to LSDB.
+
+    .. code-block:: bash
+
+        $ conda config --add channels conda-forge
+        $ conda install healpy
+        $ git clone https://github.com/astronomy-commons/lsdb
+        $ cd lsdb
+        $ pip install -e .
+
+    When installing dev dependencies, make sure to include the single quotes.
+
+    .. code-block:: bash
+
+        $ pip install -e '.[dev]'
+
+Testing
+-------------------------------------------------------------------------------
+
+Please add or update unit tests for all changes made to the codebase. You can run
+unit tests locally simply with:
+
+.. code-block:: bash
+
+    pytest
+
+If you're making changes to the sphinx documentation (anything under ``docs``),
+you can build the documentation locally with a command like:
+
+.. code-block:: bash
+
+    cd docs
+    make html
+
+Create your PR
+-------------------------------------------------------------------------------
+
+Please use PR best practices, and get someone to review your code. We have a suite
```

Comment:
Maybe say who should review it? One of our team? List some usernames for someone to add?